server/src/main.rs

```diff
@@ -209,10 +209,10 @@ struct Cli {
     #[clap(short, long, env = "DATABASE_PATH", default_value = "mycelial.db")]
     database_path: String,
 
-    #[clap(long, env = "AUTH0_AUTHORITY")]
+    #[clap(long, env = "AUTH0_AUTHORITY", default_value = "")]
```

Comment:
default values are probably not the best solution here. We might want to just wrap those args in feature flag